common/web/lm-worker/build.sh

```diff
@@ -105,6 +105,6 @@ function do_test() {
 }
 
 builder_run_action configure  verify_npm_setup
-builder_run_action clean      rm -rf build/
+builder_run_action clean      rm -rf build/ & rm -rf intermediate/
```

Comment:
Two things wrong here:

1. Single `&` operator backgrounds the first statement
2. The operator applies to `builder_run_action`, not `rm`, so even replacing with `&&` would be wrong -- it would run `rm -rf intermediate/` every time(assuming `builder_run_action` didn't fail).

Fortunately, `rm` allows for multiple parameters:

```suggestion
builder_run_action clean      rm -rf build/ intermediate/
```